lms/djangoapps/courseware/views.py

```diff
@@ -729,8 +721,6 @@ def course_info(request, course_id):
             'studio_url': studio_url,
             'show_enroll_banner': show_enroll_banner,
             'url_to_enroll': url_to_enroll,
-            'sneakpeek_allowed': sneakpeek_allowed,
```

Comment:
Restore `sneakpeek_allowed` and `regularly_registered (i.e is not lazy and is authenticated)`